src/main/java/org/prebid/server/cache/CoreCacheService.java

```diff
@@ -515,10 +536,21 @@ private static String resolveVideoBidUuid(String uuid, String hbCacheId) {
     }
 
     private void updateCreativeMetrics(String accountId, List<CachedCreative> cachedCreatives) {
-        for (final CachedCreative cachedCreative : cachedCreatives) {
+        for (CachedCreative cachedCreative : cachedCreatives) {
+            final MetricName creativeType = resolveCreativeTypeName(cachedCreative.getPayload());
+            final Integer creativeTtl = cachedCreative.getPayload().getTtlseconds() != null
+                    ? cachedCreative.getPayload().getTtlseconds()
+                    : cachedCreative.getPayload().getExpiry();
```

Comment:
We tend to avoid duplicating long method call chains. This can be simplified to smth like:

```java

final Payload payload = cachedCreative.getPayload();
final Integer creativeTtl = ObjectUtil.defaultIfNull(payload.getTtlSeconds(), payload.getExpiry());
```